src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,17 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  comments.length
+    ? (<div className="CommentList">
+      {comments.map(comment => (
+        <CommentInfo
+          comment={comment}
+          key={comment.id}
+        />
+      ))}
+    </div>)
+    : <p data-cy="NoCommentsMessage">
+      NoCommentsMessage
```

Comment:
It looks like bug 
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/70009205/e191ccef-5d1a-43e5-b3ce-a2d79a4713a3)
```suggestion
      No comments
```